<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinearCongruenceEquationCalculatorWidgetClass</class>
 <widget class="QWidget" name="LinearCongruenceEquationCalculatorWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">线性同余方程</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ax≡b(mod n)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string notr="true">a=</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditA">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string notr="true">0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string notr="true">b=</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditB">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string notr="true">0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string notr="true">n=</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditN">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string notr="true">0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string notr="true">解！</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="KatexViewer" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KatexViewer</class>
   <extends>QWidget</extends>
   <header>widgets/katexviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
